model/database/spans.yaml

```
@@ -951,6 +952,8 @@ groups:
         note: |
           `db.namespace` SHOULD be set to the combination of instance name, database name and
           service name following the `{instance_name}|{database_name}|{service_name}` pattern.
```

since you're already changing this note, mind changing the order here to 

```suggestion
          service name following the `{service_name}|{database_name}|{instance_name}` pattern.
```

We'll figure the `database_name` separately in https://github.com/open-telemetry/semantic-conventions/issues/2147